server.js

```diff
@@ -1,6 +1,18 @@
 const express = require('express');
 const app = express();
 const port = 3000; 
+const mongoose = require('mongoose');
+const URI = 'mongodb+srv://somuyak:%40Somuya2004@cluster0.i2b8prd.mongodb.net/?retryWrites=true&w=majority'
+
+
+app.get('/', (req, res) => {
```

Comment:
The database connection should be established once when the application starts, not on every request to the home route. This can lead to performance issues. [important]